project-settings/organizations/organization-settings.md

```diff
@@ -33,75 +33,82 @@ The Security tab is only available for the **Plus plan** or above
 
-![](<../../.gitbook/assets/image (620).png>)
-
-{% hint style="warning" %}
-Forest Admin supports SAML v2 (not v1)
-{% endhint %}
-
+![](<../../.gitbook/assets/organization-settings-sso-1.png>)
 
-Then choose how you want to communicate information from your Identity Provider (IP):
+| Setting | Description | Value |
+| --- | --- | --- |
+| Callback URL (Assertion Consumer Service URL)* | Assertion Consumer Service URL is responsible for receiving the SAML response | `https://api.forestadmin.com/api/saml/callback` |
+| Sign on URL* | Sign on URL | `https://api.forestadmin.com/api/saml/callback` |
+| Single Logout URL | Redirected to this location after logout | `https://app.forestadmin.com/login` (⚠️ Be careful if you use custom domains)|
+| Audience (EntityID) | Named SP Entity ID in Forest Admin | *Value is displayed in the Forest Admin* |
 
-#### method 1: XML file upload or XML file endpoint
+#### Configure Forest Admin with the Identity Provider Metadata
+**XML file upload or XML file endpoint (Recommended)**
 
 Either upload a file containing the authentication information (you'll be able to generate this file in your Identify Provider) or input the endpoint at which such a file is available (some IPs provide this).
 
 ![](<../../.gitbook/assets/image (59).png>)
 
-#### method 2: Manual input
+**Manual input**
 
 You may also enter your authentication information manually. You'll need to provide:
 
-* a login endpoint
-* a logout endpoint
-* one certificate
+* a **Login endpoint**
+* a **Logout endpoint**
+* and finally **a valid certificate**
 
 ![](<../../.gitbook/assets/image (595).png>)
 
-Click on Test configuration to try to authenticate. If it works, you're all set but you will still need to enable that new SSO authentication method:
+Click on Test configuration to try to authenticate.
+
+
+#### Enables it for all the users
+If it works, you're all set but you will still need to enable that new SSO authentication method:
 
 ![](<../../.gitbook/assets/image (289).png>)
 
 {% hint style="danger" %}
 After enabling SSO, all users will be required to [log in](./#how-to-log-in-using-single-sign-on-sso) again.
 {% endhint %}
 
-#### IDP-initiated login
+#### IDP-initiated login (Optional)
 
 Once you have enabled SSO, you have the option to enable IDP-initiated login: this will allow your users to be automatically logged in when they come to Forest Admin from your identity provider dashboard.
 
 ![](<../../.gitbook/assets/image (318).png>)
 
-To set it up properly, you will need to set a default **Relay state** on your identity provider following this format:
+To set it up properly, you will need to set a default **Relay state** on your identity provider following this format (*or URL encoded depending on the IdP*):
 
 ```javascript
-{"organizationName": "<organization_name>", "destinationUrl": "organization.projects"}
+{
+  "organizationName": "<OrganizationName_TO_REPLACE>",
```

Comment:
I am not sure we need TO_REPLACE since it should have been generated by the server with the right name